### DISCUSSION TITLE
Should we release 0.70.8?

### DISCUSSION THREAD

**Discussion by cipolleschi** [reactwg/react-native-releases#56] [Patches]
## Released, please refer to #62 

Current Release: [0.70.7](https://github.com/facebook/react-native/releases/tag/v0.70.7)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by mikeduminy** [comment]
Hey there, there is an issue in Hermes that prevents one from loading large, complex bundles in development. The change is to simply increase the max size of the stack.

The change hasn't been merged yet but it targets the stable branch within Hermes for 0.70. My guess is some coordination needs to happen to first merge this then bump this in RN. I'm happy to help where ever needed.

https://github.com/facebook/hermes/pull/924

**Reply by kelset** [reply]
hey @mikeduminy just an headsup, in order to produce a quick patch release with the fix for xcode 14.3, we had to skip this - but I'll add a comment into the next patch discussion so that we can get back to it at a less "firefight mode" time.

**Comment by cipolleschi** [comment]
Remove Ruby requirements:
- https://github.com/facebook/react-native/commit/e57b6d11fa39aa96d92a7a6b8c91429974bef81c

**Reply by kelset** [reply]
(already in the branch)

**Comment by kelset** [comment]
samsung lists + textinput issue: [this week a fix should land](https://github.com/facebook/react-native/issues/35936#issuecomment-1464830542)

> The larger change to minimize the number of spans emitted by React Native is currently being validated and is targeting next week to hit the main branch.

We should wait for it before doing a new patch.

**Comment by fortmarek** [comment]
[Prevent native blob resource from being de-allocated prematurely](https://github.com/reactwg/react-native-releases/discussions/59#discussioncomment-5321454): 
- https://github.com/facebook/react-native/commit/36cc71ab36aac5e5a78f2fbae44583d1df9c3cef

**Comment by cipolleschi** [comment]
Cherry-pick, fix for Xcode 14.3.0: 
- https://github.com/facebook/react-native/pull/36759

Changes probably need to be applied manually (no cherry-picked), due to Monorepo changes.

**Reply by kelset** [reply]
https://github.com/facebook/react-native/pull/36768
